docs/architecture.md

```diff
@@ -0,0 +1,60 @@
+# Architecture
```

Comment:
Moved and renamed that file to "headless-chrome.md". It didn't contain anything but headless Chrome stuff :)